src/Common/Color.h

```diff
@@ -0,0 +1,67 @@
+/******************************************************************************
```

Comment:
My opinion is that having a class like this is OOP overkill. I would just have some constants header with 
```
using Color = uint32_t;
namespace colors
{
constexpr Color k_White = 0xFFFFFFFF;
constexpr Color k_Black = 0x000000FF;
constexpr Color k_Good = 0xEBEBB7FF
constexpr Color k_Evil = 0xFFB4B4FF;
}; // namespace colors
```
Use comments if the element ordering needs to be clarified.

Reply:
Are we going to be doing a 1 to 1 reimplantation of LH3DColor?